dev/com.ibm.ws.messaging.common/resources/com/ibm/ws/sib/processor/CWSIPMessages.nlsprops

```diff
@@ -1069,9 +1069,9 @@ NO_NEIGHBOUR_REPLY_WARNING_CWSIP0381.useraction=Make sure that the remote messag
 
 # -------------------------------------------------------------------------------------------------
 # {0} The UUID of the Remote messaging engine
-NEIGHBOUR_REPLY_RECIEVED_INFO_CWSIP0382=CWSIP0382I: Messaging engine {0} responded to subscription request, Publish Subscribe topology now consistent.
-NEIGHBOUR_REPLY_RECIEVED_INFO_CWSIP0382.explanation=Successful communications have been made between messaging engines and the Publish subscribe Topology is consistent.
-NEIGHBOUR_REPLY_RECIEVED_INFO_CWSIP0382.useraction=none
+NEIGHBOUR_REPLY_RECEIVED_INFO_CWSIP0382=CWSIP0382I: Messaging engine {0} responded to subscription request, Publish Subscribe topology now consistent.
```

Comment:
Suggested edit:
The {0} messaging engine responded to a subscription request. The  publish subscribe topology is now consistent.

Reply:
I'm not comfortable changing this current user visible message. If you feel strongly I'd say we should open an issue with the team that owns this message to ensure the message they are trying to convey is still correct with any additions.

Reply:
understood, I'll approve based on your current edits